docs/sdk/server-side-sdks/python/python-gettingstarted.md

```diff
@@ -8,22 +8,44 @@ sidebar_custom_props: {icon: rocket}
 [![PyPI](https://badgen.net/pypi/v/devcycle-python-server-sdk)](https://pypi.org/project/devcycle-python-server-sdk/)
 [![GitHub](https://img.shields.io/github/stars/devcyclehq/python-server-sdk.svg?style=social&label=Star&maxAge=2592000)](https://github.com/DevCycleHQ/python-server-sdk)
 
-## Initializing SDK 
+## Initializing Local Bucketing SDK
+
+Code sample for importing and setting up the DevCycleLocalClient.
+
+```python
+    from devcycle_python_sdk import DevCycleLocalClient, DevCycleLocalOptions
+    from devcycle_python_sdk.models.user import DevCycleUser
+    
+    # Create an options object to do custom configurations, or use the defaults
+    options = DevCycleLocalOptions()
+    
+    # create an instance of the DevCycleCloudClient class
+    client = DevCycleLocalClient('YOUR_DVC_SERVER_SDK_KEY', options)
```

Comment:
```suggestion
    client = DevCycleLocalClient('YOUR_DEVCYCLE_SERVER_SDK_KEY', options)
```

Reply:
personally, no i don't think so.  seems like we have a habit of sprinkling that in here and there but aren't consistent.  I'll remove it.